components/home/Banner.tsx

```diff
@@ -70,26 +70,26 @@ function MobileBanner() {
     <Flex
       justify={'end'}
       direction={'column'}
-      w={'24.125rem'}
-      h={'23.25rem'}
+      w={'21rem'}
```

Comment:
Let's start using chakra numbers wherever possible. `w={5}` rather than `w={'21rem'}`. If you need 21 rem & chakra doesn't provide it by default, you can extend the chakra config and add a new number with 21rem & use it. This will make it standard